README.md

```diff
@@ -14,6 +14,57 @@ This SDK current supports the following versions of CloudEvents:
 
 Package **cloudevents** provides primitives to work with CloudEvents specification: https://github.com/cloudevents/spec.
 
+Creating minimal CloudEvents with data in version 1.0.0:
+
+### Binary HTTP CloudEvent
+
+```python
+from cloudevents.sdk.http_events import CloudEvent
+import requests
+
+
+# This data defines a binary cloudevent
+headers = {
+    "Content-Type": "application/json",
+    "ce-specversion": "1.0",
+    "ce-type": "README.sample.binary",
+    "ce-id": "binary-event",
+    "ce-time": "2018-10-23T12:28:22.4579346Z",
+    "ce-source": "README",
+}
+data = {"message": "Hello World!"}
+
+event = CloudEvent(data, headers=headers)
+headers, body = event.to_request()
+
+# POST
+requests.post("<some-url>", json=body, headers=headers)
+```
+
+### Structured HTTP CloudEvent
+
+```python
+from cloudevents.sdk.http_events import CloudEvent
+import requests
+
+
+# This data defines a structured cloudevent
+data = {
+    "specversion": "1.0",
+    "type": "README.sample.structured",
+    "id": "structured-event",
+    "source": "README",
+    "data": {"message": "Hello World!"}
+}
+event = CloudEvent(data)
+headers, body = event.to_request()
+
+# POST
+requests.post("<some-url>", json=body, headers=headers)
+```
```

Comment:
Sending CloudEvents should be under receiving CloudEvents in the README as it is relatively less common.

Reply:
Unsure of where exactly you want me to move this code sample in the README

Reply:
Do you want me to move this section further down in the README?

Reply:
I think we should follow-up in a different PR to add a clear way to create a server handler that receives a CE request.